lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1913,33 +1913,39 @@ def present(self, state):
 
         # If the image parameter was passed then we need to deal with the image
```

Comment:
I think the comment needs updating to match the new behavior?

Reply:
Indeed! I've updated it.